### PR TITLE
Add stats default fields (hostname, http_address, version)

### PR DESCRIPTION
**Pull request by andrewvc**
Backport #5520 to 5.0 branch.

### CODE REVIEW

File: logstash-core/lib/logstash/api/commands/base.rb [hunk 1]
```diff
@@ -8,14 +8,10 @@ def initialize(service = LogStash::Api::Service.instance)
...
 
-        def hostname
-          service.agent.node_name
-        end
-
         def uptime
           service.agent.uptime
         end
-
+
```
Comment (suyograo) [inline logstash-core/lib/logstash/api/commands/base.rb]: extra whitespace?

File: logstash-core/spec/api/spec_helper.rb [hunk 2]
```diff
@@ -92,7 +94,7 @@ def do_request(&block)
     @runner = LogStashRunner.new
     @runner.start
   end
-
+
```
Comment (suyograo) [inline logstash-core/spec/api/spec_helper.rb]: whitespace?

**Comment by suyograo** [comment]
@andrewvc are the test failures legit? Its not about publishing a snapshot, no?

**Comment by andrewvc** [comment]
Merging this directly since tests use snapshot gems in the 5.0 branch.

Tests passed locally.

File: logstash-core/spec/api/spec_helper.rb (hunk 2 above)
Comment (andrewvc) [inline logstash-core/spec/api/spec_helper.rb]: the whitespace things here just bring the whitespace to the correct point, indented with blacks.

**Comment by suyograo** [comment]
🚢

**Comment by suyograo** [comment]
LGTM

**Pull request merged**